### DISCUSSION TITLE
Popup menu for the QSO record at the placeholder of the search result

### DISCUSSION THREAD

**Discussion by R1BLH** [magicbug/Cloudlog#2809] [General]
When searching for a QSO by callsign, the placeholder of the search result does not allow you to display a popup menu of actions with the found record of the QSO.

![2023-12-12_01-04-15](https://github.com/magicbug/Cloudlog/assets/16667173/89767e39-36ff-4838-a8aa-af6425778f2a)

For example, the screenshot shows the search result with only one line. In this case, the entire popup menu is displayed inside the placeholder of the search result, so it is not available to the user without scrolling inside the placeholder.

![2023-12-12_01-05-05](https://github.com/magicbug/Cloudlog/assets/16667173/75466022-72a9-4c15-8cd2-eada96798d7d)
![2023-12-12_01-24-32](https://github.com/magicbug/Cloudlog/assets/16667173/a86f2aa3-23b6-4e48-a43e-e5e71854735f)

**Comment by HB9HIL** [comment]
We know about this issue and trying to fix this. Since we upgraded to Bootstrap 5 this is a pretty tricky one. If you have any suggestions we would appreciate that 😂👍

**Reply by R1BLH** [reply]
Oops... my hands are shaped for soldering iron, not for web programming :)
I have only one idea: is it possible to set the height of the placeholder manually? If this is possible, then calculate the necessary size (height of the popup menu plus height of the search results table).

**Reply by HB9HIL** [reply]
we already tried that. We will find a solution :) thank you for reporting that 👍
